_dev/apps/ui/src/store/modules/product-feed/mutations.ts

```diff
@@ -201,4 +200,15 @@ export default {
   ) {
     state.report.productsInCatalog = productsInCatalog;
   },
+  [MutationsTypes.SAVE_VERIFICATION_STATS](
+    state: LocalState,
+    verificationStats: {
+      scannedProducts: number;
+      invalidProducts: number;
+    },
+  ) {
+    state.report.nonCompliantProducts = verificationStats.invalidProducts;
+    state.report.verifiedProducts = verificationStats.scannedProducts
```

Comment:
SI c'est les produits valides, ce n'est pas les produits vérifiés. Je trouve que ça serait plus claire compliantProducts / validProducts / validatedProducts

Reply:
De plus, si c'est la données nécessaire en Front, ça serait peut etre plus simple d'envoyer les bonnes infos depuis le back non ?

Reply:
> Si c'est les produits valides, ce n'est pas les produits vérifiés. Je trouve que ça serait plus claire compliantProducts / validProducts / validatedProducts

In that case, let's wait for the ongoing wording review. I'm pretty sure we'll have some changes to apply which can be done in another PR.

> De plus, si c'est la données nécessaire en Front, ça serait peut etre plus simple d'envoyer les bonnes infos depuis le back non ?

Who may eventually do this and when? You or @SachaAvazashvili?